cyclops-ui/src/components/k8s-resources/common/PodTable/PodManifest.tsx

```diff
@@ -0,0 +1,138 @@
+import { CopyOutlined } from "@ant-design/icons";
+import { Alert, Button, Checkbox, Col, Modal, Tooltip } from "antd";
+import { CheckboxChangeEvent } from "antd/es/checkbox";
+import axios from "axios";
+import { useCallback, useEffect, useState } from "react";
+import ReactAce from "react-ace/lib/ace";
+
+interface PodManifestProps {
+  pod: any;
+}
+
+const PodManifest = ({ pod }: PodManifestProps) => {
+  const [manifest, setManifest] = useState("");
+  const [showManagedFields, setShowManagedFields] = useState(false);
+  const [modal, setModal] = useState({
+    on: false,
+  });
+  const [error, setError] = useState({
+    message: "",
+    description: "",
+  });
+
+  const closeModal = () => {
+    setModal({
+      on: false,
+    });
+    setManifest("");
+  };
+
+  const fetchManifest = useCallback(async () => {
+    if (modal.on) {
+      const { group, name, namespace } = pod;
+      const { data: manifestData } = await axios.get(`/api/manifest`, {
+        params: {
+          group,
+          version: "v1",
+          kind: "Pod",
+          name,
+          namespace: namespace.length ? namespace : "default",
+          includeManagedFields: showManagedFields,
+        },
+      });
+      setManifest(manifestData);
+    }
+  }, [pod, showManagedFields, modal]);
+
+  useEffect(() => {
+    fetchManifest();
+  }, [showManagedFields, fetchManifest]);
+
+  const handleCheckboxChange = (e: CheckboxChangeEvent) => {
+    setShowManagedFields(e.target.checked);
+  };
+
+  return (
+    <Col span={12} style={{ paddingLeft: 4 }}>
+      <Button
+        style={{ width: "100%" }}
+        onClick={() => {
+          setModal({ on: true });
+        }}
+      >
+        View Manifest
```

Comment:
Can you make this `<h4>`. Also, you can add this icon since other buttons have it - `<FileTextOutlined />`